<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>582</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="1" rowspan="2" colspan="7">
      <widget class="QSlider" name="horizontalSliderMin">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="4" rowspan="3">
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>58</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="10" column="0" rowspan="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Timing(s)</string>
       </property>
      </widget>
     </item>
     <item row="8" column="8" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Max</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QPushButton" name="pushButtonStop">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item row="3" column="9" rowspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Min</string>
       </property>
      </widget>
     </item>
     <item row="12" column="4">
      <widget class="QPushButton" name="pushButtonStart">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item row="5" column="8" colspan="3">
      <widget class="QLCDNumber" name="lcdNumberMin">
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
     <item row="7" column="1" rowspan="2" colspan="7">
      <widget class="QSlider" name="horizontalSliderMax">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="11" rowspan="14">
      <widget class="QListWidget" name="listWidget"/>
     </item>
     <item row="2" column="6">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>48</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="9" column="8" colspan="3">
      <widget class="QLCDNumber" name="lcdNumberMax">
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="9">
      <widget class="QLineEdit" name="lineEdit_IP"/>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="pushButtonConnect">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item row="2" column="7">
      <widget class="QPushButton" name="pushButtonDisconect">
       <property name="text">
        <string>Disconnect</string>
       </property>
      </widget>
     </item>
     <item row="11" column="6">
      <spacer name="verticalSpacer_7">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>28</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="11" column="2">
      <spacer name="verticalSpacer_8">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>28</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="10" column="1" rowspan="2" colspan="6">
      <widget class="QSlider" name="horizontalSliderTiming">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="10" column="9" rowspan="2" colspan="2">
      <widget class="QLabel" name="label_Timing">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>623</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSliderMin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberMin</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderMax</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberMax</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderTiming</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_Timing</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
